Makefile

```diff
@@ -25,13 +25,16 @@ release: clean npm $(BUILD_DIR)/public/js copy sass js
 ## dev: create a debug build for a platform in build/$BROWSER/dev
 ## specify browser=(chrome|chrome-mv3|firefox) type=dev
 dev: $(BUILD_DIR)/public/js copy sass js
+ifndef question
+	echo $(shell date "+%s") > $(BUILD_DIR)/buildtime.txt
```

Comment:
I wanted to make `$(BUILD_DIR)/buildtime.txt` a proper build target, then give that the same dependencies as the `dev` target, then add `$(BUILD_DIR)/buildtime.txt` as a dependency to `dev`. But it seems that since the `copy`, `sass` and `js` targets are marked `.PHONY` they always run, so the `buildtime.txt` would always be updated.

I'm thinking that perhaps we should assign the dependencies for `sass`, `copy` and `js` to variables, then use those variables for the dependencies of `dev` + `release` instead. I think that way, the dependencies wouldn't be phony, and we could use those for the dependencies of the `$(BUILD_DIR)/buildtime.txt` target too.

What do you think?

Reply:
Sounds fine to me as long we're not using them outside of the makefile.